test/include/nntrainer_test_util.h

```diff
@@ -91,6 +92,10 @@ class IniSection {
 
   IniSection operator+(const std::string &s) { return IniSection(*this) += s; }
 
+  void rename(const std::string &name) { section_name = name; }
```

Comment:
FYI, in 4f17acd, I have made SectionName only ctor, so it can be done like
`IniSection("block2") + backbone_valid` to provide isolation between each section.

Reply:
I tried this but it failed, didn't investigate, will update in a later PR.